src/machines/systemIO/systemIOMachineDesktop.ts

```diff
@@ -24,6 +24,11 @@ import {
 } from '@src/machines/systemIO/utils'
 import { fromPromise } from 'xstate'
 import type { AppMachineContext } from '@src/lib/types'
+import {
```

Comment:
Updated SystemIOMachine to have the same workflows as the FileMachine which is now deleted. These are required for generic application workflows as well as the project explorer.